compiler/src/iree/compiler/Codegen/SPIRV/SPIRVVectorizeLoadStore.cpp

```diff
@@ -52,6 +54,18 @@ static bool getUsesIfAllTransferOp(Value value,
                  << "failed: non-transfer-like user: " << *userOp << "\n");
       return false;
     }
+    bool isMinorIdentity = true;
+    if (auto readOp = dyn_cast<vector::TransferReadOp>(userOp)) {
```

Comment:
nit: you can merge those two with `VectorTransferOpInterface`

Reply:
Good point. Changed.